core/services/chainlink/config_dump.go

```diff
@@ -652,7 +652,6 @@ func (c *Config) loadLegacyCoreEnv() {
 			Frequency:        envDuration("DatabaseBackupFrequency"),
 			Mode:             legacy.DatabaseBackupModeEnvVar.ParsePtr(),
 			OnVersionUpgrade: envvar.NewBool("DatabaseBackupOnVersionUpgrade").ParsePtr(),
-			URL:              envURL("DatabaseBackupDir"),
```

Comment:
Will it dump to a secrets.toml instead?

Reply:
I omitted secrets.toml from the dump command by design, since it seemed unecessary (e.g. they may want to stick to env vars only) and potentially risky. Do you think this is important to include?

Reply:
I was also under the impression that we never want to dump any secrets.

Reply:
Agree should not be dumpable